src/utils/substr.ts

```diff
@@ -1,8 +1,8 @@
-function substr(address: string) {
+function substr(address: string, value = 4) {
```

Comment:
Just one more suggestion
```suggestion
function substr(address: string, indexEnd= 4) {
```
```suggestion
function substr(address: string, length= 4) {
```